tfx/orchestration/kubeflow/kubeflow_dag_runner.py

```diff
@@ -25,12 +26,15 @@
 from tfx import version
 from tfx.dsl.compiler import compiler as tfx_compiler
 from tfx.dsl.components.base import base_component as tfx_base_component
+from tfx.dsl.components.base import base_node
 from tfx.orchestration import data_types
 from tfx.orchestration import pipeline as tfx_pipeline
 from tfx.orchestration import tfx_runner
 from tfx.orchestration.config import pipeline_config
 from tfx.orchestration.kubeflow import base_component
+from tfx.orchestration.kubeflow import utils
 from tfx.orchestration.kubeflow.proto import kubeflow_pb2
+from tfx.orchestration.kubeflow.v2.compiler_utils import TFX_DAG_NAME
```

Comment:
this name is better defined in the common file (eg.  tfx/orchestration/kubeflow/utils.py) instead imported from v2 folder.

If you don't want to include change in current pull, I can refactor later. 
Just heads up here.

Reply:
That's fair. I'm happy to include it. I was a bit wary of adding too many changes for this

Reply:
you can define same name and import it on compile_utils, I can make change imports of v2 to avoid making the pull too big.